commands/init/mod.ts

```diff
@@ -95,24 +96,37 @@ export const init = command("init", {
     );
 
     const existingLinks = await config.get("projects");
-    let app: paths["/projects/{id}"]["get"]["responses"]["200"]["content"][
+    let app: paths["/apps/{id}"]["get"]["responses"]["200"]["content"][
       "application/json"
     ];
     const link = existingLinks[dest];
 
     if (!link) {
       logger.info(`Project doesn't exist. Creating a new one.`);
-      const res = await loading(projects.create({ name: appName }), {
-        text: "Creating app",
-        enabled: !flags.json,
-      });
+      const res = await loading(
+        apps.create({
+          config: {
+            apiVersion: "latest",
+            image: "some demo image",
```

Comment:
What should we use here if we should be hitting this endpoint?

Reply:
i think we need to prompt for any extra input, such as image, machine type, etc.